modules/UI/videolayout/SmallVideo.js

```diff
@@ -512,6 +513,18 @@ export default class SmallVideo {
         if (this.displayMode !== oldDisplayMode) {
             logger.debug(`Displaying ${displayModeString} for ${this.id}, data: [${JSON.stringify(displayModeInput)}]`);
         }
+
+        if (this.displayMode !== DISPLAY_VIDEO
+            && this.displayMode !== DISPLAY_VIDEO_WITH_NAME
+            && displayModeInput.tileViewActive
+            && displayModeInput.isScreenSharing
+            && !displayModeInput.isAudioOnly) {
```

Comment:
Why do we need the not audioOnly mode check here ? The user can be in audioOnly mode and in tile view and still experience this issue where screenshare isn't displayed. Am I missing something ?

Reply:
This code detects only what we show in the thumbnail. It looks like by design the thumbnail doesn't show the video of SS while we are in tile view if audio only mode is enabled.

Reply:
Oh I didn't realize that we don't show SS in tile view when in audio only mode. Thanks for clarifying.

Reply:
I wonder why it was implemented that way though ?